index.html

```diff
@@ -12,13 +12,24 @@
 </head>
 
 <body>
-    <div id="app">
+<div id="app">
         <about></about>
-        <navbar v-bind:suites="suites" v-if="loaded"></navbar>
-        <template v-for="(suite,index) in suites">
-            <task-suite v-bind:suite="suite" v-bind:index="index"></task-suite>
-        </template>
-    </div>
+        <div class="r_flex_container">
+            <div class="r_flex_fixed_child_top">
+                <navbar v-bind:suites="suites" v-if="loaded"></navbar>
+            </div>
+            
+            <div class="r_flex_expand_child">
```

Comment:
are these divs needed out of components? If possible, add them as part of the component or as separate components.

If it is too hard, complex, or impossible, just leave them as they are, It is not a big problem

Reply:
I think there are neededs D: